test/images/iperf/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM fedora
```

Comment:
Oh you have a MAINTAINER, just the license then
eg https://github.com/kubernetes/kubernetes/blob/master/test/images/n-way-http/Dockerfile